### PR TITLE
Add missing class descriptions to the phase field module

### PR DESCRIPTION
**Pull request by dschwen**
This adds missing class descriptions to the phase field module. I'll look at tensor mechanics next...

Refs #5195

### CODE REVIEW

**Comment by dschwen** [comment]
@cpritam is reviewing this. Do not merge yet.

**Comment by dschwen** [comment]
This is ready to be merged now.

File: modules/phase_field/src/materials/LinearIsoElasticPFDamage.C [hunk 1]
```diff
@@ -10,8 +10,11 @@ template<>
 InputParameters validParams<LinearIsoElasticPFDamage>()
 {
   InputParameters params = validParams<LinearElasticMaterial>();
+  params.addClassDescription("Phase-field fracture model energy contribution to damage growth");
   params.addRequiredCoupledVar("c","Order parameter for damage");
   params.addParam<Real>("kdamage",1e-6,"Stiffness of damaged matrix");
+  params.addClassDescription("Strain energy density in damaged isotropic elastic material - undamaged stress under compressive strain");
```
Comment (permcody) [inline modules/phase_field/src/materials/LinearIsoElasticPFDamage.C]: Two descriptions in the same class?

File: modules/tensor_mechanics/src/kernels/StressDivergencePFFracTensors.C [hunk 2]
```diff
@@ -5,8 +5,10 @@ template<>
 InputParameters validParams<StressDivergencePFFracTensors>()
 {
   InputParameters params = validParams<StressDivergenceTensors>();
+  params.addClassDescription("Phase field fracture model stress divergence kernel");
   params.addCoupledVar("c", "Phase field damage variable: Used to indicate calculation of Off Diagonal Jacobian term");
   params.addParam<std::string>("pff_jac_prop_name","","Name of property variable containing d_stress_d_c");
+  params.addClassDescription("Stress divergence kernel for phase-field fracture that additionally computes off diagonal damage dependent Jacobian components");
```
Comment (permcody) [inline modules/tensor_mechanics/src/kernels/StressDivergencePFFracTensors.C]: Again, two class descriptions?

**Pull request merged**